lib/ProductOpener/Display.pm

```diff
@@ -6212,7 +6213,8 @@ HTML
 <nav class="tab-bar show-for-small-only">
 	<div class="left-small" style="padding-top:4px;">
 		<a href="#idOfLeftMenu" role="button" aria-controls="idOfLeftMenu" aria-expanded="false" class="left-off-canvas-toggle button postfix">
-		<i class="icon-account_box" style="color:$torso_color;font-size:1.8rem"></i></a>
+		<span style="color:$torso_color;font-size:1.8rem">@{[ display_icon('account_box') ]}</span>
```

Comment:
@hangy same here for `font-size`.

We could add:
```scss
#idOfLeftMenu {
  .icon {
    font-size: 1.8rem;
  }
}
```
to `_off.scss`

Reply:
I agree that we could move this inline style to the SCSS file, but the element with id `idOfLeftMenu` doesn't appear to actually exist on the page. I wonder how the expand button actually works in this case.